cli/pkg/local/git.go

```diff
@@ -256,30 +256,12 @@ func (s *Server) GitPush(ctx context.Context, r *connect.Request[localv1.GitPush
 	return connect.NewResponse(&localv1.GitPushResponse{}), nil
 }
 
-func inferRillManagedProjectName(ctx context.Context, h *cmdutil.Helper, org, pathToProject string) (string, error) {
+func inferProject(ctx context.Context, h *cmdutil.Helper, org, pathToProject string) (*adminv1.Project, error) {
```

Comment:
This method feels overkill. Can we just replace it with call to `cmdutil.Helper`'s method?